orchestrator/gravity_proto/src/ethereum_claim.rs

```diff
@@ -0,0 +1,135 @@
+use deep_space::Address;
+
+/// A mirror of the EthereumClaim interface on the Go side
+/// EthereumClaim represents a claim on ethereum state
+pub trait EthereumClaim {
```

Comment:
this isn't auto-generated I'm not really sure it belongs in proto

Reply:
Should I move it to gravity_utils then? Or cosmos_gravity? I was a bit confused on the placement.

Reply:
utils may be better but it could go either way.